docs/developers/meta/sc-config.md

```diff
@@ -82,7 +88,38 @@ The settings are as follows:
     - _default_: `[]`
 - `kill-legacy-callback`
     - The framework has no way of knowing whether or not a certain smart contract variant actually needs the async callback code or not, so in rare situations it is necessary to forcefully remove it.
-    - _values_: `true` | `false
+    - _values_: `true` | `false`
+    - _default_: `false`
+- `codegen-units`
+    -  Controls the number of "code generation units" a crate will be split into. Splitting a crate into multiple code generation units can have a significant impact on the compile time and code optimization of the crate.
+    - _default_: `1`
+- `opt-level`
+    - Controls the level of optimization that Rust will apply to the code during compilation.
+    - _values_:
+        - `0`: no optimizations;
+        - `1`: basic optimizations;
+        - `2`: some optimizations;
+        - `3`: all optimizations;
+        - `s`: optimize for binary size;
+        - `z`: optimize for binary size, but also turn off loop vectorization.
+    - _default_: `z`
+- `lto`
+    - Enables link-time optimization for the release profile.
+    - _values_: `true` | `false`
+    - _default_: `true`
+- `debug`
+    - Controls the amount of debug information included in the compiled binary.
+    - _values_: `true` | `false`
+    - _default_: `false`
+- `panic`
+    - Controls how smart contracts handles panics, which are unexpected errors that occur at runtime.
+    - _values_:
```

Comment:
Add:
- using "unwind" is not tested as it makes little sense in a smart contract.